jdaviz/core/validunits.py

```diff
@@ -73,12 +73,12 @@ def create_flux_equivalencies_list(flux_unit, spectral_axis_unit):
 
     # Get local flux units.
     locally_defined_flux_units = ['Jy', 'mJy', 'uJy', 'MJy',
-                                  'W / (Hz m2)',
-                                  'eV / (s m2 Hz)',
-                                  'erg / (s cm2 Hz)',
-                                  'erg / (s cm2 Angstrom)',
+                                  'W / (m2 Hz)', 'W / (Hz m2)',  # Order is different in astropy v5.3  # noqa
+                                  'eV / (s m2 Hz)', 'eV / (Hz s m2)',
+                                  'erg / (s cm2 Angstrom)', 'erg / (s cm2 Angstrom)',
+                                  'erg / (s cm2 Hz)', 'erg / (Hz s cm2)',
                                   'ph / (Angstrom s cm2)',
-                                  'ph / (Hz s cm2)',
+                                  'ph / (Hz s cm2)', 'ph / (Hz s cm2)'
```

Comment:
These two are also identical rather than reversed.

Reply:
I had removed these units before and needed to find the correct order of the denominator units, I should have all the duplicates and the change in orders resolved in the commit I am about to push. I have kept the change in order from astropy v5.3 changes, I don't know if its still relevant to keep both, its something I can investigate and test out removing one or the other.

Reply:
I don't think you have to worry about astropy version below this pin:

https://github.com/spacetelescope/jdaviz/blob/d71dc9157a79b5ded5d22441b3bb71cdd668a6a2/pyproject.toml#L10